etc/mpv.desktop

```diff
@@ -26,11 +45,21 @@ Comment[da]=Afspil film og sange
 Comment[de]=Filme und Musik abspielen
 Comment[es]=Reproduzca vídeos y canciones
 Comment[fr]=Lire des vidéos et des musiques
+Comment[fr_BE]=Lecture de films et de chansons
+Comment[hi]=फिल्में और गाने चलाएं
+Comment[hu]=Játssz filmeket és dalokat
+Comment[it]=Lettore multimediale
```

Comment:
duplicate of existing line ;p

Reply:
What is the language that is duplicated. If I made a mistake, I will try to correct it. Please be more specific.
The languages "pt" and "pt_BR" are different, as are "fr" and "fr_BE", in case you didn't know that. I can speak in the "pt_BR" language, PPC and Zeh in the "pt" language.

- - - - -

Qual é o idioma qu está duplicado. Se eu errei, eu tentarei corrigir. Por favor, seja mais específico.
Os idiomas "pt" e "pt_BR" são diferentes, assim como "fr" e "fr_BE", caso você não saiba disso. Eu posso falar pelo idioma "pt_BR" o PPC e o Zeh pelo idioma "pt".

Reply:
Two lines below is the same line.

Reply:
I still can't see what the duplicate line is.

- - - - -

Eu ainda não consigo ver qual é a linha duplicada.

Reply:
`Comment[it]=Lettore multimediale`

Reply:
Thanks t-8ch, now we are speaking the same "language".
Now I need to figure out how I can edit again to remove the repeated entry.
Sorry for this error.

- - - - -

Obrigado t-8ch, agora estamos falando a mesma "língua".
Agora eu preciso descobrir como poderei editar novamente para remover a entrada repetida.
Desculpe-me por este erro.

Reply:
I was able to edit the file and remove the "Comment[it]=Lettore multimediale" entry that was repeated.

- - - - -

Eu consegui editar o arquivo e remover a entrada "Comment[it]=Lettore multimediale" que estava repetida.

Reply:
A few tips for the contribution:

* Use better commit messages
* Squash your fixups into the existing commit to avoid unnecessary commits
* Don't open multiple PRs for the same change, if you do mention it and maybe close the old one
* If the project maintainers tell you that the name is "mpv", respect that. Especially as now the name is even inconsistent within mpv.desktop.

Reply:
t-8ch

A few tips for the contribution:
R: Sobre as suas orientações.

"Use better commit messages"
R: Eu preciso de um modelo ou um exemplo, pois eu comecei a pouco a operar o GitHub e GitLab. Estas duas plataformas possuem algo em comum, ambas são confusas, não são intuitivas e parecem que foram feitas para serem difíceis.

"Don't open multiple PRs for the same change, if you do mention it and maybe close the old one"
Eu posso tentar fazer isso, assim eu aprendo com vocês.

"If the project maintainers tell you that the name is "mpv", respect .that. Especially as now the name is even inconsistent within mpv.desktop."
A escrita com letras maiúsculas foi a que eu encontrei no arquivo .desktop que está no antiX desde maio de 2020, foi quando eu tive o meu primeiro contado com o mpv. Se está sendo empacotado com letra maiúscula, eu imagino que seja porque não há traduções no "mpv.desktop" e o empacotador do .deb fez o que pode com o que tinha nas mãos. 
Eu respeito a escrita com letras minúsculas, apesar de tornar os menus feios e sem o apelo natural que uma letra maiúscula causa nos olhos do observador. Eu respeito a escrita com letras minúsculas até o limite de não ferir a regra ortográfica do meu idioma, pois o que vale para mim é a norma ortográfica do meu idioma. Eu respeito o seu idioma e isso deve ser recíproco para o meu idioma. Por exemplo, não iniciamos as frases com letras minúsculas, isso é uma regra e não tem o que ser questionada.

Eu agradeço por sua atenção e por tentar me explicar o que eu devo fazer aqui no GitHub.

Meus cordiais cumprimentos.

Reply:
Note: I'm not a mpv maintainer or affiliated in any way with the project. I'm just sharing my experience contributing to this and lots of other projects.

Yes, there is no course on how to use these platforms. So it makes sense to take the guidance that is given by regular users of the platform to heart. If it seems like the platforms are made intentionally hard to use please consider that they may be optimized for different usecases or processes than you are expecting.

The fact that antiX had its own version of the desktop file for some time does not really matter for the upstream mpv project.

As for the spelling not being the usual for a given language: The antiX project itself has some usual spelling.
Is the spelling adapted for the different languages or is it always the same?

Please also consider that while it is absolutely true that Linux distros should be usable by users in all kinds of languages, this here is the issuetracker of the mpv project. So the local language conventions should be followed.

In general if you are contributing to some projects you should always follow the local guidelines and patterns.
Consistency is very important in collaborative environments, otherwise it will devolve into an incomprehensible mess.

Reply:
Please t-8ch could you use the texts from the file "arandr.desktop.txt"?
I changed the capital letters to lowercase letters in the writing of "mpv".
Sorry for the inconvenience.

[arandr.desktop.txt](https://github.com/mpv-player/mpv/files/9574926/arandr.desktop.txt)

- - - - - -

Por favor, t-8ch você poderia utilizar os textos do arquivo "arandr.desktop.txt"?
Eu alterei as letras letras maiúsculas por letras minúsculas na escrita de "mpv".
Desculpe-me pelo transtorno.

Reply:
You should add these changes to your branch. I can't write to it.
Then try to squash all commits together [0] and change the commitmessage to something meaningful like
"mpv.desktop: add more translations".

After that ask the maintainers again to take a look at this PR.
(Also close your other PR for the same changes with a pointer to this PR)

[0] https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git